docs/get-started/backup/backup_mongo.md

```diff
@@ -0,0 +1,22 @@
+# Create and restore MongoDB backups
```

Comment:
In this page we're providing only requirements and prerequisites. Let's name it just **MongoDB backups** or similar.